jeecloud.exception.service.alreadyExists=Service with name "{0}" and URL "{1}" already exists
jeecloud.exception.service.notExists=Service with name "{0}" does not exists